scripts/jenkins_pr_describe.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Can we delete this script now (since we have `createPrDescription` in the Jenkinsfile)?

Reply:
deleted